src/libcore/marker.rs

```diff
@@ -29,6 +29,7 @@ use crate::hash::Hasher;
 /// [arc]: ../../std/sync/struct.Arc.html
 /// [ub]: ../../reference/behavior-considered-undefined.html
 #[stable(feature = "rust1", since = "1.0.0")]
+#[cfg_attr(not(test), rustc_diagnostic_item = "send")]
```

Comment:
```suggestion
#[cfg_attr(not(test), rustc_diagnostic_item = "send_trait")]
```
(more clear what it refers to)

Reply:
Fixed!